src/services/reporting/index.ts

```diff
@@ -0,0 +1,44 @@
+import { createApiPathWithValidate } from "../../lib/path"
+import { WithValidationError } from "../../index"
+import { ApiCallOptions, ignoreServerSideErrors, postData } from "../../base"
+
+// eslint-disable-next-line @typescript-eslint/ban-types
+export const postTrackingData = async <T extends {}>({
```

Comment:
I looked into accepting both types but this leads into other issues as typescript complains about the response to potentially be of the wrong type as the specific call only allows of one of the types in the response